guides/release/pages.yml

```diff
@@ -1,325 +1,325 @@
 - title: "Introduction"
   url: "toc-heading_introduction"
   is_heading: true
-- title: "Guides and Tutorials"
+- title: "Guides et Tutoriels"
   url: "index"
   skip_toc: true
   pages:
-    - title: "Ember.js Guides"
+    - title: "Guides Ember.js"
       url: ""
-- title: "Getting Started"
+- title: "Commencer"
   url: "getting-started"
   pages:
-    - title: "How To Use The Guides"
+    - title: "Utiliser le Guide"
       url: "index"
-    - title: "Quick Start"
+    - title: "Démarrage rapide"
       url: "quick-start"
-    - title: "Working with HTML, CSS, and JavaScript"
+    - title: "Travailler avec HTML, CSS, et JavaScript"
       url: "working-with-html-css-and-javascript"
-    - title: "Anatomy of an Ember App"
+    - title: "Anatomie d'une App Ember"
       url: "anatomy-of-an-ember-app"
-- title: "Tutorial"
+- title: "Tutoriel"
   url: "tutorial"
   pages:
-    - title: "Part 1"
+    - title: "Partie 1"
       url: "part-1"
       pages:
         - title: "Introduction"
           url: "index"
         - title: "Orientation"
```

Comment:
Oh, right, I think I didn't translate this one because I had no idea and I just forgot about it XD
Maybe we can just completely move out from the initial wording here.
```suggestion
        - title: "Créer une App"
```